src/udev/udev-builtin-net_id.c

```diff
@@ -1172,17 +1183,7 @@ static int builtin_net_id(UdevEvent *event, int argc, char *argv[], bool test) {
 
         udev_builtin_add_property(dev, test, "ID_NET_NAMING_SCHEME", naming_scheme()->name);
 
-        if (names_mac(dev, &info) >= 0) {
-                char str[ALTIFNAMSIZ];
-
-                xsprintf(str, "%sx%s", prefix, HW_ADDR_TO_STR_FULL(&info.hw_addr, HW_ADDR_TO_STRING_NO_COLON));
-                udev_builtin_add_property(dev, test, "ID_NET_NAME_MAC", str);
-                log_device_debug(dev, "MAC address identifier: hw_addr=%s %s %s",
-                                 HW_ADDR_TO_STR(&info.hw_addr),
-                                 special_glyph(SPECIAL_GLYPH_ARROW_RIGHT), str + strlen(prefix));
-
-                ieee_oui(dev, &info, test);
-        }
+        (void) names_mac(dev, prefix, test);
```

Comment:
If the return value is not needed, why don't you make the function `void`? (Same for the following commits.)

Reply:
I just want to use `return log_device_debug_errno(...);`.